apps/cyberstorm-remix/app/p/packageVersion.tsx

```diff
@@ -0,0 +1,545 @@
+                >
+                  <Await resolve={version}>
+                    {(resolvedValue) => (
+                      <>{packageMeta(firstUploaded, resolvedValue)}</>
+                    )}
+                  </Await>
+                </Suspense>
+              </div>
+            </aside>
+          </div>
+        </section>
+      </div>
+    </>
+  );
+}
+
+const Actions = memo(function Actions(props: {
+  team: Awaited<ReturnType<typeof getTeamDetails>>;
+  version: Awaited<ReturnType<typeof getPackageVersionDetails>>;
+}) {
+  const { team, version } = props;
+  return (
+    <div className="package-listing-sidebar__actions">
+      <NewButton
+        primitiveType="link"
+        href={version.download_url}
+        csVariant="secondary"
+        rootClasses="package-listing-sidebar__download"
+      >
+        <NewIcon csMode="inline" noWrapper>
+          <FontAwesomeIcon icon={faDownload} />
+        </NewIcon>
+        Download
+      </NewButton>
+      {team.donation_link ? (
+        <NewButton
+          primitiveType="link"
+          href={team.donation_link}
+          csVariant="secondary"
+          csSize="big"
+          csModifiers={["only-icon"]}
+        >
+          <NewIcon csMode="inline" noWrapper>
+            <FontAwesomeIcon icon={faHandHoldingHeart} />
+          </NewIcon>
+        </NewButton>
+      ) : null}
+    </div>
+  );
+});
+
+function packageMeta(
+  firstUploaded: ReactElement | undefined,
+  version: Awaited<ReturnType<typeof getPackageVersionDetails>>
+) {
+  return (
+    <div className="package-listing-sidebar__meta">
+      <div className="package-listing-sidebar__item">
+        <div className="package-listing-sidebar__label">Date Uploaded</div>
+        <div className="package-listing-sidebar__content">{firstUploaded}</div>
+      </div>
+      <div className="package-listing-sidebar__item">
+        <div className="package-listing-sidebar__label">Downloads</div>
+        <div className="package-listing-sidebar__content">
+          {formatInteger(version.download_count)}
+        </div>
+      </div>
+      <div className="package-listing-sidebar__item">
+        <div className="package-listing-sidebar__label">Size</div>
+        <div className="package-listing-sidebar__content">
+          {formatFileSize(version.size)}
+        </div>
+      </div>
+      <div className="package-listing-sidebar__item">
+        <div className="package-listing-sidebar__label">Dependency string</div>
+        <div className="package-listing-sidebar__content">
+          <div className="package-listing-sidebar__dependency-string-wrapper">
+            <span
+              title={version.full_version_name}
+              className="package-listing-sidebar__dependency-string"
+            >
+              {version.full_version_name}
+            </span>
+            <CopyButton text={version.full_version_name} />
+          </div>
+        </div>
+      </div>
+    </div>
+  );
+}
```

Comment:
This component and thepackageVersionWithoutCommunity should also be checked for unnecessary duplicate code. If it can't reasonably be reduced, please explain why.